scipy/signal/windows/_windows.py

```diff
@@ -2273,6 +2482,11 @@ def get_window(window, Nx, fftbins=True):
         `ifftshift` and be multiplied by the result of an FFT (see also
         :func:`~scipy.fft.fftfreq`).
         If False, create a "symmetric" window, for use in filter design.
+    xp : module, optional
+        Optional array API namespace. defaults to NumPy.
```

Comment:
Could you change this parameter in each function to something like

```suggestion
    xp : array_namespace, optional
        Optional array namespace.
        Should be compatible with the array API standard, or supported by array-api-compat.
        Default: ``numpy``
```